src/witan/app/handler.clj

```diff
@@ -23,7 +23,7 @@
 
 (def LoginReturn
   (s/either {(s/required-key :token) s/Str
-             (s/required-key :id) s/Str}
+             (s/required-key :id) s/Uuid}
```

Comment:
And yes, this now curls properly for me and also the tests are adjusted to use UUIDs and they pass.